### ISSUE TITLE
problem with static images of tablesorter

### ISSUE THREAD

**Issue by gustavo-salazar** [ebiwd/EBI-Framework#150]
Hey there,
I just tried the latest version 1.3.10 an I'm going to have to revert because there is an asset with a wrong path, and it is giving me errors when compiling all my css files.

The error I found is in https://github.com/ebiwd/EBI-Framework/blob/v1.3/css/ebi-global.css#L5256 
Seems that the Library where the image is have included a `dist` folder, that it's missing in the path.

**Comment by khawkins98** [comment]
Hi. Sounds like you're using the npm version — could you give me an example of which asset(s) are missing?

**Comment by khawkins98** [comment]
Hi again. I've published `1.3.11`, this reinstates some dependencies that somehow didn't get published last time. 

Shout if it it doesn't sort your issue?

**Comment by gustavo-salazar** [comment]
Unfortunately this second release doesn't solve the issue. So let me try to explain myself better.
* We use the npm package.
* From it, we import the `ebi-framework/css/ebi-global.css`
* The line [5256](https://github.com/ebiwd/EBI-Framework/blob/v1.3/css/ebi-global.css#L5256) of ebi-global.css is:
```
  background-image: url(../libraries/tablesorter/css/images/black-asc.gif); 
```
* However that image in the version 1.3.11 of the package is in `/libraries/tablesorter/dist/css/images/black-asc.gif`. Notice the **/dist**. You can see it in this [CDN](https://cdn.jsdelivr.net/npm/ebi-framework@1.3.11/libraries/tablesorter/dist/css/images/).
* So when in our code we process the CSS files, webpack gives an error because it can find the image.
* This wasn't the case in [1.3.8](https://cdn.jsdelivr.net/npm/ebi-framework@1.3.8/libraries/tablesorter/css/images/).

Thanks for your help,
Gustavo.

**Comment by khawkins98** [comment]
Finally figured this one out, there was a new setting in tablesorter's pacakage.json that was resulting in only the `/dist` directory being tracked. 

I've npm-ignored it and all seems well now https://github.com/ebiwd/EBI-Framework/commit/3e58d87a08d4003e5a533bffe1d0fe8d0ec882aa

```bash
C02PQ23ZFVH6:test khawkins$ npm init -y
Wrote to /Users/khawkins/Documents/Git/test/package.json:

C02PQ23ZFVH6:test khawkins$ npm install ebi-framework
npm WARN test@1.0.0 No description
npm WARN test@1.0.0 No repository field.

+ ebi-framework@1.3.13
updated 1 package and audited 1 package in 7.331s
found 0 vulnerabilities

C02PQ23ZFVH6:test khawkins$ cd node_modules/ebi-framework/libraries/tablesorter/css/
bootstrap.less         high-specificity/      psd/                   theme.bootstrap.css    theme.default.css      theme.grey.css         theme.less             
dragtable.mod.css      images/                theme.black-ice.css    theme.bootstrap_2.css  theme.dropbox.css      theme.ice.css          theme.metro-dark.css   
filter.formatter.css   metro.less             theme.blue.css         theme.dark.css         theme.green.css        theme.jui.css          

```

**Comment by khawkins98** [comment]
Aside: present-me would tell past-me to have npm-installed tablesorter instead, but we're in minimal effort maintenance of 1.3 while we focus on 2.0

**Comment by gustavo-salazar** [comment]
Thanks a lot @khawkins98 I'll give it a try but from what you are showing here that should fix the issue

**Comment by gustavo-salazar** [comment]
Yep, just confirming this release, fix my issue. 
Thanks so much.

BTW, you have no idea how much I relate with the present-me vs past-me comment! Good luck with v2.0

**Comment by gustavo-salazar** [comment]
Hey @khawkins98,
It seems the same error is happening for version 1.3.14

https://cdn.jsdelivr.net/npm/ebi-framework@1.3.14/libraries/tablesorter/css/images/

**Comment by khawkins98** [comment]
Finally got to this.

Have published 1.3.16 that sorts this again:
- https://cdn.jsdelivr.net/npm/ebi-framework@1.3.16/libraries/tablesorter/css/images/

Root problem is Tablersorter's `package.json` which needs to be deleted for npm publishing to correctly work:
- https://github.com/ebiwd/EBI-Framework/commit/78fa9f8d2c4dc8bcb738486e17908c6d2c3defbf

Hopefully we won't see it crop back up.
